pkg/ovs/openflow/ofctrl_bridge.go

```diff
@@ -119,9 +137,11 @@ type OFBridge struct {
 
 func (b *OFBridge) CreateTable(id, next TableIDType, missAction MissActionType) Table {
 	t := newOFTable(id, next, missAction)
+
 	b.Lock()
```

Comment:
Only one goroutine works on creating the oftables, so there should be no concurrency.